sql/core/src/test/resources/sql-functions/sql-expression-schema.md

```diff
@@ -206,9 +204,6 @@
 | org.apache.spark.sql.catalyst.expressions.Pow | power | SELECT power(2, 3) | struct<POWER(CAST(2 AS DOUBLE), CAST(3 AS DOUBLE)):double> |
 | org.apache.spark.sql.catalyst.expressions.Quarter | quarter | SELECT quarter('2016-08-31') | struct<quarter(CAST(2016-08-31 AS DATE)):int> |
 | org.apache.spark.sql.catalyst.expressions.RLike | rlike | SELECT '%SystemDrive%\Users\John' rlike '%SystemDrive%\\Users.*' | struct<%SystemDrive%UsersJohn RLIKE %SystemDrive%\Users.*:boolean> |
-| org.apache.spark.sql.catalyst.expressions.Rand | random | SELECT random() | struct<rand():double> |
-| org.apache.spark.sql.catalyst.expressions.Rand | rand | SELECT rand() | struct<rand():double> |
-| org.apache.spark.sql.catalyst.expressions.Randn | randn | SELECT randn() | struct<randn():double> |
```

Comment:
I think @beliefer meant https://github.com/apache/spark/pull/28392 is only landed into master and we should keep these just as are at https://github.com/apache/spark/pull/28194. I assume this seed were printed out because this test was ran against the latest `master` instead of `branch-3.0` at https://github.com/apache/spark/pull/28427/commits/948a54e7059b29a1a001315b0ed2d69ec4104314

Reply:
Yea, I think this golden file was generated in master, so I re-generated it in branch-3.0. Btw, we need this test suite in branch-3.0? Reverting #28194 from branch-3.0 is another simple solution to recover it, I think.

Reply:
This test is useful to track the auto-alias name change history. It would be nice to have it in 3.0.